specs/spring-cloud-contract-spec-java/src/main/java/org/springframework/cloud/contract/spec/internal/RegexPatterns.java

```diff
@@ -27,9 +27,12 @@
  * @author Tim Ysewyn
  * @since 1.0.0
  */
-public class RegexPatterns {
+public final class RegexPatterns {
+
+	public RegexPatterns() {
+		System.out.println("WARNING: RegexPatterns shouldn't be instantiated");
```

Comment:
`private` constructor and throw an exception upon instantiation instead of doing `sout`